src/test/java/org/fcrepo/camel/integration/FcrepoDeleteIT.java

```diff
@@ -182,15 +185,15 @@ public void testDeleteNestedContainer() throws InterruptedException {
         deletedEndpoint.assertIsSatisfied();
 
         // Check deleted container
-        for (Exchange exchange : goneEndpoint.getExchanges()) {
-            Assert.assertTrue(exchange.getIn().getHeader(Exchange.CONTENT_TYPE, String.class).contains("text/html"));
-            Assert.assertTrue(exchange.getIn().getBody(String.class).contains("Gone"));
-        }
-
-        for (Exchange exchange : notFoundEndpoint.getExchanges()) {
-            Assert.assertTrue(exchange.getIn().getHeader(Exchange.CONTENT_TYPE, String.class).contains("text/html"));
-            Assert.assertTrue(exchange.getIn().getBody(String.class).contains("Not Found"));
-        }
+        goneEndpoint.getExchanges().forEach(exchange -> {
+            assertTrue(exchange.getIn().getHeader(Exchange.CONTENT_TYPE, String.class).contains("application/rdf+xml"));
+            assertTrue(exchange.getIn().getBody(String.class).contains("Discovered tombstone"));
+        });
+
+        notFoundEndpoint.getExchanges().forEach(exchange -> {
+            assertTrue(exchange.getIn().getHeader(Exchange.CONTENT_TYPE, String.class).contains("text/html"));
+            assertTrue(exchange.getIn().getBody(String.class).contains("Not Found"));
```

Comment:
What I'm saying is this: I'm not going to remove all of those blocks right now. We can either wait until next week (or later) when I _could_ remove those blocks, and have this PR sit until then, or we can remove them as some future, unspecified point. Either way, I'm not removing them today. If anyone wants to make a PR on PR to remove them, I'd be supportive of that.